docs/source/conf.py

```diff
@@ -41,8 +41,12 @@
     "myst_parser",
     "notfound.extension",
     "sphinxcontrib.jquery",
+    'sphinx_favicon',
 ]
 
+favicons = [
+    "https://kedro.org/images/favicon.ico",
```

Comment:
@Huongg Is it possible to add favicon locally from repo instead of downloading from https://kedro.org/images/favicon.ico

Reply:
hey @jitu5  it's possible but it means we have to do the same download for framework and kedro-datasets, at the moment 3 repos are using the same image from the link

Reply:
I feel having it in one place for all the repos is easy for maintenance. Also, where are we actually downloading this favicon, here we are just defining the source. Do you know where is the download code ? or does sphinx do it internally

Reply:
If we are already following this then its fine. I just asked because if the link somehow gets broken, it will break everywhere, and we won't know about it.

Reply:
@ravi-kumar-pilla i dont think it's just accessing the link, so the proper fix will be to get all of this handled inside kedro-sphinx-theme but it requires a lot of investigation i think. Juanlu has already opened the ticket for it so hopefully we get it fixed soon